packages/assets-controllers/src/NftController.ts

```diff
@@ -1081,10 +1099,11 @@ export class NftController extends BaseController<NftConfig, NftState> {
     });
   }
 
-  async validateWatchNft(
+  private async validateWatchNft(
```

Comment:
Do we want to use the newer private syntax?
```suggestion
 async #validateWatchNft(
```

Reply:
I had it this way originally but all the other private methods on this controller use `private` so I thought I'd follow that pattern and we could swap them all together at a later time.